package/ebs/rancher-ebs

```diff
@@ -282,7 +282,25 @@ attach() {
     fi
 
     if [ "$linux_device_path" != "" ]; then
-        print_device ${linux_device_path}
+        mountpoint="/var/lib/rancher/volumes/rancher-ebs/${OPTS[name]}-staging"
+        mkdir -p ${mountpoint}
+        error=`mount ${linux_device_path} ${mountpoint} 2>&1`
+        if [ $? -ne 0 ]; then
+            if [ -z "$(echo $error | grep 'is already mounted')" ]; then
+                print_error $error
+            fi
+        fi
+        cd ${mountpoint}
+        if [ -z "$(ls -A | grep -v lost+found | grep -v _data | grep -v .rancher-v1 | grep -v .rancher-v2)" ]; then
+            rm -r .rancher-v1 > /dev/null 2>&1
+            touch .rancher-v2 > /dev/null 2>&1
+            mountdest="${mountpoint}/_data"
+            mkdir -p ${mountdest}
+        else
+            rm -r .rancher-v2 > /dev/null 2>&1
+            touch .rancher-v1 > /dev/null 2>&1
+        fi
+        print_device $linux_device_path
```

Comment:
The intention is that in v2 mode, this function returns the_rancherdata dir as the device which is later supplied to the mountdest as $DEVICE, so v2 vs v1 mode should be transparent to the mountdest